packages/common/src/services.ts

```diff
@@ -721,6 +721,34 @@ export class CondaPackage implements Conda.IPackageManager {
     }
   }
 
+  async getDependencies(
+    pkg: string,
+    cancellable = true
+  ): Promise<Conda.IPackageDeps> {
+    this._cancelTasks();
+    const request: RequestInit = {
+      method: 'GET'
+    };
+
+    const { promise, cancel } = Private.requestServer(
+      URLExt.join('conda', 'packages') +
+      URLExt.objectToQueryString({ package: pkg }),
+      request
+    );
+
+    let idx: number;
+    if (cancellable) {
+      idx = this._cancellableStack.push(cancel) - 1;
```

Comment:
Follow-up on cancellation changes:

```suggestion
      idx = this._cancellableStack.push({type: 'getDependencies', cancel}) - 1;
```